flixel/math/FlxPoint.hx

```diff
@@ -96,6 +98,20 @@ class FlxPoint implements IFlxPooled
 	}
 	
 	/**
+	 * Set the coordinates of this point (inline).
+	 * 
+	 * @param	X	The X-coordinate of the point in space.
+	 * @param	Y	The Y-coordinate of the point in space.
+	 * @return	This point.
+	 */
+	public inline function setInline(X:Float, Y:Float):FlxPoint
```

Comment:
It's definitely noticeable in my profiles, although we could omit this in favor of direct inline variable access (via ```_x``` and ```_y```).